File: eFormAPI/Plugins/Vehicles.Pn/Vehicles.Pn/Models/VehiclesPnRequestModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Vehicles.Pn.Models
{
    public class VehiclesPnRequestModel
    {
        public string SortColumnName { get; set; }
        public int PageIndex { get; set; }
        public int Offset { get; set; }
        public int PageSize { get; set; }
        public bool IsSortDsc { get; set; }
    }
}

File: eform-client/src/app/plugins/modules/vehicles-pn/models/vehicles-pn-request.model.ts
export class VehiclesPnRequestModel {
  sortColumnName: string;
  pageIndex: number;
  offset: number;
  pageSize: number;
  isSortDsc: boolean;

  constructor() {
    this.sortColumnName = 'id';
    this.isSortDsc = true;
    this.pageSize = 10;
    this.pageIndex = 1;
    this.offset = 0;
  }
}
